src/components/engagement/dto/pnp-data.dto.ts

```diff
@@ -16,4 +16,10 @@ export abstract class PnpData {
 
   @Field(() => Float)
   variance: number;
+
+  @Field(() => Number)
```

Comment:
graphql has floats and ints, no numbers.
```suggestion
  @Field(() => Int)
```